main.go

```diff
@@ -137,8 +160,17 @@ func collectCarthageCache(projectDir string) error {
 	if err != nil {
 		return fmt.Errorf("failed to determine cache paths")
 	}
+
 	carthageCache := cache.New()
+
+	// Cachefile
 	carthageCache.IncludePath(fmt.Sprintf("%s -> %s", absCarthageDir, absCacheFilePth))
+
+	// Cartfile.resolved
+	resolvedFilePath := filepath.Join(projectDir, resolvedFileName)
+	carthageCache.IncludePath(resolvedFilePath)
```

Comment:
why do we need to add the resolvedFilePath to the cache archive?
the cache indicator file should contain the resolved file's content already.